README.md

```diff
@@ -23,16 +23,16 @@ documentation].
 
 | Release |  Current Patch  | Release Date |      EOL      |
```

Comment:
```suggestion
| Release |  Current Patch  | Minor Release Date |      EOL      |
```
```suggestion
| Release |  Release Date | Current Patch  |      EOL      |
```

I got confused the last time.

Reply:
@muvaf are these two options for changing or are you thinking it should read `Releases | Minor Release Date | Current Patch | EOL`? I think I would be most amenable to just the second one: `Releases | Release Date | Current Patch | EOL`, which would just be swapping the two middle columns -- what do you think?

Reply:
I provided them as different alternatives. The second one reads better to me, too.

Reply:
Awesome! Thanks :)